src/react/account/AccountHead.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+import { Head, HeadCenter, HeadLeft, HeadRight, HeadTitle, IconCircle} from '../ui-elements/ListLayout';
```

Comment:
```suggestion
import { Head, HeadCenter, HeadLeft, HeadRight, HeadTitle, IconCircle } from '../ui-elements/ListLayout';
```

Reply:
Thank you, I fixed it and added a linter rule to enforce it :)